<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateImgDlg</class>
 <widget class="QDialog" name="CreateImgDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>381</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
    <item>
     <widget class="QGroupBox" name="gbxSize">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>72</height>
       </size>
      </property>
      <property name="title">
       <string>Size</string>
      </property>
      <widget class="QLabel" name="lblWidth">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>78</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Width:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edtWidth">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>71</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>512</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblHeight">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>78</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Height:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edtHeight">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>30</y>
         <width>91</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>512</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gbxType">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>160</height>
       </size>
      </property>
      <property name="title">
       <string>Type</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>361</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="majorTypeLayout">
        <item>
         <widget class="QRadioButton" name="btnGray">
          <property name="text">
           <string>Gray</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="btnColor">
          <property name="text">
           <string>Color</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>361</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="subTypeLayout">
        <item>
         <widget class="QGroupBox" name="gbxGray">
          <property name="title">
           <string/>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>0</y>
             <width>151</width>
             <height>71</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="chkGray8">
              <property name="text">
               <string>8-bit</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chkGray16">
              <property name="text">
               <string>16-bit</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gbxColor">
          <property name="title">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gbxVar">
      <property name="title">
       <string>Variable</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <widget class="QLabel" name="lblName">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>78</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edtVarName">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>281</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Image</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateImgDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateImgDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
